### PR TITLE
update tutorials/modules for metatensor

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

### Description
update to use metatensor

### Status
**Ready**

### Checks

- [ ] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

**Comment by wyli** [comment]
Hi @Nic-Ma @yiheng-wang-nv, I've review all notebooks in `modules/` (except `dynunet_pipeline` and `bundle`). please help review these changes.

File: modules/nifti_read_example.ipynb [hunk 1]
```diff
@@ -187,27 +196,23 @@
...
     "        RandSpatialCrop((64, 64, 64), random_size=False),\n",
-    "        EnsureType(),\n",
     "    ]\n",
     ")\n",
     "\n",
     "ds = ArrayDataset(images, imtrans, segs, segtrans)\n",
     "\n",
     "loader = torch.utils.data.DataLoader(\n",
-    "    ds, batch_size=10, num_workers=2, pin_memory=torch.cuda.is_available()\n",
+    "    ds, batch_size=10, num_workers=0, pin_memory=torch.cuda.is_available()\n",
```
Comment (Nic-Ma) [inline modules/nifti_read_example.ipynb:215]: May I know why change the `num_workers` here?

Thanks.

File: modules/nifti_read_example.ipynb [hunk 2]
```diff
@@ -255,7 +260,7 @@
     "ds = GridPatchDataset(ds, img_seg_iter, with_coordinates=False)\n",
     "\n",
     "loader = torch.utils.data.DataLoader(\n",
-    "    ds, batch_size=10, num_workers=0, pin_memory=torch.cuda.is_available()\n",
+    "    ds, batch_size=10, num_workers=2, pin_memory=torch.cuda.is_available()\n",
```
Comment (Nic-Ma) [inline modules/nifti_read_example.ipynb:263]: Same as above comments.

Thanks.

**Review by Nic-Ma — APPROVED** [review]
Thanks for the update.
Looks good to me, put some question inline.

**Pull request merged**